app/constants/post_draft.js

```diff
@@ -8,7 +10,7 @@ export const ICON_SIZE = 24;
 export const INSERT_TO_COMMENT = 'insert_to_comment';
 export const INSERT_TO_DRAFT = 'insert_to_draft';
 export const IS_REACTION_REGEX = /(^\+:([^:\s]*):)$/i;
-export const MAX_FILE_COUNT = 5;
+export const MAX_FILE_COUNT = isMinimumServerVersion(Client4.serverVersion, 5, 32) ? 10 : 5;
```

Comment:
this does not really works as this constant is set as soon as the file is imported, this can happen way before the `Client4.serverVersion` is set, perhaps this should be changed from a constant to a helper function instead.

Reply:
Thanks @enahum. I created the helper function and replaced the constant wherever I could find it. However I wasn't sure where to place it so I created a new file.